Dedicated standard and advanced tiers (#16068)
cockroachdb/docs: cockroachcloud/plan-your-cluster.md

All {{ site.data.products.db }} clusters use 3 Availability Zones (AZs). For balanced data distribution and best performance, we recommend using a number of nodes that is a multiple of 3 (e.g., 3, 6, or 9 nodes per region).

#### {{ site.data.products.dedicated }} advanced

> You should choose {{ site.data.products.dedicated }} advanced if your cluster needs access to all features required for [PCI compliance](../{{site.versions["stable"]}}/security-reference/security-overview.html). {{ site.data.products.dedicated }} advanced clusters have all the same features as {{ site.data.products.dedicated }} base clusters with the addition of these security features.

Reviewer: "base clusters" change to "standard clusters"

> You should choose {{ site.data.products.dedicated }} advanced if your cluster needs access to all features required for [PCI readiness](../{{site.versions["stable"]}}/security-reference/security-overview.html). {{ site.data.products.dedicated }} advanced clusters have all the same features as {{ site.data.products.dedicated }} standard clusters with the addition of these security features.

Reviewer 2: ```suggestion
You should choose {{ site.data.products.dedicated }} advanced if your cluster needs access to all features required for [PCI readiness](../{{site.versions["stable"]}}/security-reference/security-overview.html). {{ site.data.products.dedicated }} advanced clusters have all the features of {{ site.data.products.dedicated }} standard clusters plus these security features.
```
